uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -18,7 +18,7 @@ define([
     '$filter', '$uibModal', '$dialog', '$timeout', 'AssignmentService', 'DocumentService', 'ContactService', 'FileService', 'data',
```

Comment:
> getting rid of $scope, private functions + vm.function, order of things
This is already fixed in #239 

I will fix the naming for controller and implement `$inject` in this PR.